src/components/HomePage/ProductPlacementArea/ProductDescription.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import productDescriptionStyle from './ProductDescription.module.css';
+
+const ProductDescription = props => {
+
+    const {setRef} = props;
+    const ref = React.useRef(null);
+    setRef(ref);
```

Comment:
Is this because of scrolling? Would it be better to use an id directly or is there some limitation in that approach?

Reply:
Refs must be used to access the html element the React way, so this is the right way

Reply:
Fair point, did not know about that! A quick search prooved me wrong, though 🤦‍♂ 
Truck away and ignore some other comments regarding this, except the one debating the multi-level prop passing (redux as another possibility?)